sdk/java/pulumi/src/main/java/com/pulumi/deployment/InvokeOptionsBuilder.java

```diff
@@ -0,0 +1,54 @@
+package com.pulumi.deployment;
+
+import com.pulumi.resources.ProviderResource;
+import com.pulumi.resources.Resource;
+import javax.annotation.Nullable;
+
+public final class InvokeOptionsBuilder {
```

Comment:
I think it's convention (at least in our SDK) to have the builder be a `static` inner class of the type it's constructing, named `Builder`. I reckon we should follow that here so we can be consistent?